### PR TITLE
Upgraded test_views

### LINKED ISSUE #120: Upgrade test_views
After views functions were updated - test_views needed updating.

### PR DESCRIPTION
**Pull request by DanWaldman**
After views.py was updated - test_views needed upgrading.
Now tests_views is more comprehensive. Closes #120

### CODE REVIEW

**Review by shakedhagag — APPROVED** [review]
Seems to me you have successfully solved the issue

**Review by razashtar93 — APPROVED** [review]
LGTM

**Comment by DanielItkes** [comment]
Nice work! test expert :heart_eyes: :heart_eyes:

**Review by DanielItkes — APPROVED** [review]

File: AcadeMeData/tests/test_views.py [hunk 1]
```diff
@@ -1,15 +1,25 @@
...
+from AcadeMeData.models import University, Degree
 
 
 @pytest.mark.django_db
-def test_homepage_html(client):
-    response = views.homePage(client)
-    assert response.content.decode() == render_to_string('landing/homepage.html')
+def test_homepage_html(client, user_example):
+    all_universities = University.objects.all()
+    all_degrees = Degree.objects.all()
```
Comment (Yarboa) [inline AcadeMeData/tests/test_views.py:9]: Consider adding fixture for here, that do the same in conftest

**Review by Yarboa — APPROVED** [review]
LGTM, there is one comment, you could decide what to do

File: AcadeMeData/tests/test_views.py (hunk 1 above)
Comment (DanWaldman) [reply AcadeMeData/tests/test_views.py]: Done

**Review by Yarboa — APPROVED** [review]
LGTM,

**Pull request merged**